### PR TITLE
Install OpenTelemetry with Sentry in Djano and add some instrumentation

### PR DESCRIPTION
**Pull request by Moggach**
## Description
This PR installs the [OpenTelemetry SDK and configures it to send data to Sentry](https://docs.sentry.io/platforms/python/performance/instrumentation/opentelemetry/)
It adds some instrumentation to the get_scheduled_batch_job_progress() function to send metrics on importing and updating external data sources
- time per job
- amount of rows per job
- failed jobs 

@janbaykara or @joaquimds it would be great to chat this through with one of you to see if I'm on the right lines. In particular how I can best send this data to Sentry as it's kind of buried in the Transaction > Summary > Event Details
https://common-knowledge-7k.sentry.io/performance/meep-intelligence-hub:2d6d54af835f4524ad6f1849388b16dc/?environment=production&environment=development&project=4506972731342848&query=&referrer=performance-transaction-summary&statsPeriod=90d&transaction=get_scheduled_batch_job_progress&unselectedSeries=p100%28%29&unselectedSeries=avg%28%29
and ideally we'd want to be getting an average of these values over many transactions

Thanks

### CODE REVIEW

File: hub/models.py [hunk 1]
```diff
@@ -2095,7 +2134,7 @@ def clear_permissions_cache_intersecting_user(sender, instance, *args, **kwargs)
         cache.delete(sharing_permission.get_cache_key())
 
 
-class Report(PolymorphicModel):
```
Comment (janbaykara) [inline hub/models.py:2098]: What is the rationale for this?

**Comment by janbaykara** [comment]
FYI we're using a different Sentry org, it's `commonknowledge` not `common-knowledge-7k`.

File: hub/models.py (hunk 1 above)
Comment (Moggach) [reply hub/models.py:2098]: None, that's a mistake. I'll fix.

**Comment by Moggach** [comment]
@janbaykara and @joaquimds 
Updated this PR to remove previous metrics on get_scheduled_batch_job_progress() and replace with the following:
Metrics to get 
- number of rows per import and update job
- duration of each import and update job
telemetry_task function that wraps tasks to record
- CPU usage per task
- task status e.g. succeeded or failed
A span to get time difference between request and task beginning on import_all task

Everything is logged to the console for the moment to enable testing and not spam Sentry

**Pull request merged**